includes/class-object-cache.php

```
@@ -96,48 +96,51 @@ public function get_status() {
+				return $status;
+			}
+
+			$status['redis_reachable'] = true;
+			$redis                     = $connection;
+
+			// Collect telemetry if enabled and reachable.
+			if ( $status['enabled'] ) {
+				$info = $redis->info();
+				if ( $info ) {
+					$status['telemetry'] = array(
+						'redis_version'          => $info['redis_version'] ?? 'Unknown',
```

PHPStan (2.1.51)

[warning] Cannot access offset 'redis_version' on non-empty-array<string, mixed>|Redis|non-falsy-string.


(offsetAccess.nonOffsetAccessible)